### DISCUSSION TITLE
list installed versions and most recently released versions of custom node packages

### DISCUSSION THREAD

**Discussion by dryodryo** [Comfy-Org/ComfyUI-Manager#1109] [Ideas]
It would be super helpful if the Manager listed the installed and most recently released versions of custom node packages. The Manager only displays the date of the most recent package update, which is not very useful to me. I have no way of knowing if installed custom node packages are up to date or not. And I am extremely wary of "Update All".

Thanks!

**Comment by ltdrdata** [comment]
I will consider this feature for a future update.

**Comment by dryodryo** [comment]
Thank you for all of your many contributions!
